source/docs/yearly-changes/yearly-changelog.rst

```diff
@@ -117,18 +135,39 @@ New Configs
 
 We've added several new configs. A full list of available configs is available in the ``configs`` (Java, C++, Python) namespace.
 
+Improved Support for roboRIO Motion Profiles
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Additional support has been added for various feedforward terms (kS, kG, kV and KA). There is now improved integration with roboRIO motion profiling using velocity or acceleration setpoints in various closed loop controls.
```

Comment:
"using velocity or acceleration setpoints in various closed loop controls"
Maybe explicitly mention velocity in position control and acceleration in velocity control?